sorting_it_out/test.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+from sorting_it_out.__main__ import sorting_it_out
+
+
+class TestSortingItOut(unittest.TestCase):
+    def test_it_should_pass_if_return_empty_result(self):
+        value = ""
+        actual = sorting_it_out(value)
+        expect = ""
+        self.assertEqual(expect, actual)
+
+    def test_it_should_pass_if_not_contain_any_non_alphabetical_characters(self):
+        value = "<,?':.abcABa"
+        actual = sorting_it_out(value)
+        expect = "aaabbc"
+        self.assertEqual(expect, actual)
+
+    def test_it_should_pass_if_not_contain_any_white_space(self):
+        value = " abcAAA"
+        actual = sorting_it_out(value)
+        expect = "aaaabc"
+        self.assertEqual(expect, actual)
+
+    def test_it_should_pass_if_return_correct_value_when_case_sensitive(self):
+        value = "ABCabc"
+        actual = sorting_it_out(value, case_sensitive=True)
+        expect = "abcABC"
+        self.assertEqual(expect, actual)
+
+    def test_it_should_pass_if_return_correct_result(self):
```

Comment:
bỏ hết mấy chữ test cho nó gọn cấu trúc là câu là it should do stuff when condition is

Reply:
chỗ này phải có prefix test á anh, ko có em chạy test ko dc